cloud/pkg/cloudstream/connection.go

```diff
@@ -63,34 +63,73 @@ type ContainerLogsConnection struct {
 	edgePeerStop chan struct{}
 }
 
+// ContainerMetricsConnection indicates the containerMetrics request initiated by kube-apiserver
+type ContainerMetricsConnection struct {
+	// MessageID indicate the unique id to create his message
+	MessageID    uint64
+	ctx          context.Context
+	r            *restful.Request
+	flush        io.Writer
+	session      *Session
+	edgePeerStop chan struct{}
+}
+
 func (l *ContainerLogsConnection) GetMessageID() uint64 {
 	return l.MessageID
 }
 
+func (ms *ContainerMetricsConnection) GetMessageID() uint64 {
```

Comment:
It is recommended that you place the two structures and their methods  in two source files.

`ContainerLogsConnection` in containerlog_connection.go
`ContainerMetricsConnection ` in containermetrics_connection.go 
?

Reply:
Done.

Reply:
@kadisi